homeassistant/components/simplisafe/__init__.py

```diff
@@ -653,6 +654,13 @@ async def async_update_system(system: SystemType) -> None:
             if isinstance(result, SimplipyError):
                 raise UpdateFailed(f"SimpliSafe error while updating: {result}")
 
+    async def async_media_request(self, url: str) -> bytes | None:
+        """Fetch a media file."""
+        content = await self._api.async_media(url)
+        if content is None:
```

Comment:
Is this necessary? Can we just `return content`?

Reply:
Because if I just "return content" mypy says:  error: Returning Any from function declared to return "bytes | None"  [no-any-return]

Reply:
And in simplisafe-python (_api) the function is declared:
  async def async_media(self, url: str) -> bytes | None:

So I don't understand why mypy is freaking out.